### ISSUE TITLE
You have too few comments, please set at least 10 distinct comments to avoid looking suspicious.

### ISSUE THREAD

**Issue by matej2** [InstaPy/InstaPy#5251]
## Expected Behavior

Quickstart would run

## Current Behavior

Hello,

When i try to run project as described [here](https://github.com/Herrox/instapy-docker/blob/master/documentation/docker.md) i get this error: 
> ERROR [2019-11-08 14:00:14] [borishocevar35210]  You have too few comments, please set at least 10 distinct comments to avoid looking suspicious.

## Possible Solution (optional)

I was looking into Instapy issues and one of them mentioned [this possible solution](https://github.com/timgrossmann/InstaPy/issues/4441#issuecomment-494371736).

## InstaPy configuration

My quickstart file includes more than 10 comments: 

```
comments = ['Nice shot! @{}',
        'I love your profile! @{}',
        'Your feed is an inspiration :thumbsup:',
        'Just incredible :open_mouth:',
        'What camera did you use @{}?',
        'Love your posts @{}',
        'Looks awesome @{}',
        'Getting inspired by you @{}',
        ':raised_hands: Yes!',
        'I can feel your passion @{} :muscle:',
        'Nice photo',
        'Cool photo']
```

At first i tought that this is the docker issue, so i submitted the issue to [instapy-docker](https://github.com/InstaPy/instapy-docker/issues/57) repo, but they said that this error seems to be related directly to InstaPy and not to Docker.

**Comment by oshi-pog** [comment]
bump

**Comment by matej2** [comment]
bump

**Comment by nalakawula** [comment]
bump

**Comment by elidrissizack** [comment]
bump

**Comment by matej2** [comment]
Really? Will this ever get resolved?
bump

**Comment by matej2** [comment]
bump

**Comment by matej2** [comment]
This is not resolved, I am opening new issue #5423
